### ISSUE TITLE
STOP CONTRADICT YOURSELF UGLY BEARD LOSER

### ISSUE THREAD

**Issue by evilkomododragon** [MonoGame/MonoGame#7711]
YOU CONTRADICT YOURSELF PIECE OF SHIT

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

.NET Core 3.1
End of Support
December 3, 2022

REMOVE THIS 

- name: Setup .NET Core SDK ${{ matrix.dotnet-version }}
        uses: actions/setup-dotnet@v1
        with:
          dotnet-version: '3.1.x'

OR I PUT THE FLYING OAK UP YOUR FUCKIN ASS

**Comment by evilkomododragon** [comment]
3.8.1 ONLY TARGETS .NET 6
NOT NET Core 3.1

PIECE OF SHIT

**Comment by Beyley** [comment]
hi we hear your concern, dotnet core 3.1 is bad, cope

**Comment by evilkomododragon** [comment]
TOM SPILMAN IS A FUCKING SISSY
HE DOESNT EVEN READ HERE ANYMORE

DUMBASSES
HE MAKES MONEY OF THE WORK YOU DO IN HIS GAME COMPANY

THAT IS THE REASON MONOGAME STILL EXISTS

**Comment by Beyley** [comment]
> MONOGAME IS ONLY HERE BECAUSE SOME GAME DEVELOPERS MAKE MONEY OF IT...

yup thats how game development with game engines works, you make money by making a game in monogame
